src/components/Button.js

```diff
@@ -1,17 +1,30 @@
-import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ name }) => (
-  <div>
-    <h1>{name}</h1>
-  </div>
-);
+export default function Button({
+  name, operator, actionBtn, onClick,
+}) {
+  return (
+    <button
+      className={`btn ${operator ? 'operatorBtn' : ''} ${
+        actionBtn ? 'actionBtn' : ''
+      }`}
+      value={name}
+      onClick={onClick}
+      type="button"
+    >
+      {name}
+    </button>
+  );
+}
+
```

Comment:
- [ ] The `+/-` button still does not work in the live version. In addition, the live version and the local version are two completely different applications. Please let's align the two.